airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcCtidInitializer.java

```diff
@@ -102,11 +102,13 @@ public static CtidGlobalStateManager getCtidInitialLoadGlobalStateManager(final
     streamsUnderVacuum.addAll(streamsUnderVacuum(database,
         ctidStreams.streamsForCtidSync(), quoteString).result());
 
-    final List<ConfiguredAirbyteStream> finalListOfStreamsToBeSyncedViaCtid =
-        streamsUnderVacuum.isEmpty() ? ctidStreams.streamsForCtidSync()
-            : ctidStreams.streamsForCtidSync().stream()
-                .filter(c -> !streamsUnderVacuum.contains(AirbyteStreamNameNamespacePair.fromConfiguredAirbyteSteam(c)))
-                .toList();
+    if (!streamsUnderVacuum.isEmpty()) {
+      throw new ConfigErrorException(
+          "Postgres database is undergoing a full vacuum - cannot proceed with the sync. Please sync again when the vacuum is finished.");
+    }
```

Comment:
Do we really want this to be a `ConfigErrorException`? Don't we want to retry syncing later on?

Reply:
Good point, will change to TransientErrorException